scipy/stats/tests/test_discrete_basic.py

```diff
@@ -48,7 +48,8 @@ def test_discrete_basic(distname, arg, first_case):
         meths = [distfn.pmf, distfn.logpmf, distfn.cdf, distfn.logcdf,
                  distfn.logsf]
         # make sure arguments are within support
-        spec_k = {'randint': 11, 'hypergeom': 4, 'bernoulli': 0, }
+        # for some distributions, this needs to be overridden
+        spec_k = {'randint': 11, 'hypergeom': 4, 'bernoulli': 0, 'wnch': 6}
         k = spec_k.get(distname, 1)
```

Comment:
I considered calling `support` to just get the support of the distributions, but this still needed special cases for distributions with infinite support. Also, the support of `wnch` technically theoretically begins at 0, but numerically the pdf is zero up to `k=6` or so, so this needed to be a special case unless I wanted to change the default params. So I left the structure of the test as-is.